### PR TITLE
MDA/MolSSI workshop page with registration

### PR DESCRIPTION
**Pull request by orbeckst**
- overview
- deadlines
- registration link
- travel information
- contact

tasks/decisions
- [x] deadline for registration
- [x] deadline for notifying bursary recipients
- [x] registration page (google form)

### CODE REVIEW

**Comment by orbeckst** [comment]
@fiona-naughton please continue working on the page.

@jennaswa I don't have any links for registration yet. Could you please look at how to write the registration section?

File: _posts/2024-05-14-ASUworkshop.md
Comment (RMeli) [inline _posts/2024-05-14-ASUworkshop.md:90]:
```suggestion
PHX airport is 5.5 miles (10 minutes by car/rideshare) from the workshop venue.
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (RMeli) [inline _posts/2024-05-14-ASUworkshop.md:97]: I'm not used to the US, but shouldn't public transports be the first suggestion/choice?

Comment (IAlibay) [reply _posts/2024-05-14-ASUworkshop.md:97]: > I'm not used to the US

Europe has entered the chat :)

I do agree that if public transport is safe & effective it should be listed first - although I also know that this isn't always the case everywhere.

File: _posts/2024-05-14-ASUworkshop.md
Comment (IAlibay) [inline _posts/2024-05-14-ASUworkshop.md:49]: Do we need to state the number of individuals the in-person meeting will be limited to?

**Review by IAlibay — APPROVED** [review]
Aside from @RMeli's comments this looks great to me!

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [reply _posts/2024-05-14-ASUworkshop.md]: I added 40 in-person.

I don't know how the hybrid portion will work, that's more @jennaswa 's domain.

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [reply _posts/2024-05-14-ASUworkshop.md]: Generally I agree with you and I re-arranged.

Public transport in Phoenix is not good _except_ when going along the Light Rail. @ianmkenney does have some interesting storied about encounters on the Light Rail at night but generally I'd consider it reasonably safe. If your destination is close to the Light Rail then use it, otherwise I'd take a car.

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [inline _posts/2024-05-14-ASUworkshop.md:72]: @IAlibay @fiona-naughton @jennaswa @armcdona we need to tell anyone who applied for a bursary asap so I added a quick turnaround for us.

Does this work?

I know that this only leaves a day for reviewing but it will be tight either way and there's not much we can do about it. 5/27 is a Thu, 5/28 is Fri.

Comment (orbeckst) [reply _posts/2024-05-14-ASUworkshop.md:72]: On May 29 I would want to tell any bursary applicants 
* if they got a bursary (travel + hotel) or just hotel,
* if they have been accepted (even if no bursary)

In the application form we should ask *(1) Will you attend only if you get travel + hotel? (2) Will you attend if you get hotel? (3) Will you attend without additional support?*

If we only informed bursary recipients that they have been accepted and not the category (3) (will attend regardless) then we make it difficult for them to book travel at reasonable cost.

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [inline _posts/2024-05-14-ASUworkshop.md:77]: We want to tell non-local applicants ASAP if they have been accepted so that they can plan travel (see above). If we wait until June 5+X to tell them, they have less than 2 weeks. The alternative is to tell _all_ non-locals to apply for a bursary, regardless of if they need it, which will add even more to our reviewing load.

We could say that application deadline for non-locals is May 27. However, then we effectively give priority to non-locals as we don't know how many locals want to come.

Therefore, I suggest that we **keep a single deadline for all applications** and we make this deadline **May 27 for everyone**. In this way we will have all information when we need it.

*If we don't get enough applications we can then extend it until June 5*.

Opinions @IAlibay @fiona-naughton @jennaswa @armcdona  ?

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [inline _posts/2024-05-14-ASUworkshop.md:57]: Do we want to make a single registration button here?

**Review by orbeckst — COMMENTED** [review]
My primary question/suggestion is to have a **single application deadline of May 27** — please see comments for rationale.

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [reply _posts/2024-05-14-ASUworkshop.md:57]: I added one that looks like this

image

File: _posts/2024-05-14-ASUworkshop.md
Comment (RMeli) [reply _posts/2024-05-14-ASUworkshop.md]: That's fair. If they are not particularly safe or effective, then it's fine to list alternative first.

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:17]:
```suggestion
(MolSSI) and with the support of the [Chan Zuckerberg Initiative][CZI] and the
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:18]:
```suggestion
[Center for Biological Physics][CBP], is holding a **free, 2-day workshop** on **June
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:42]:
```suggestion
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:43]:
```suggestion
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:53]:
```suggestion
accommodation costs for participants from outside the [greater Phoenix area](https://en.wikipedia.org/wiki/Phoenix_metropolitan_area). Bursaries
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:58]:
```suggestion
<a href="https://forms.gle/UeiFJ6rbnXnhosnWA" target="_blank" style="background:#FF9200;padding:10px;margin:10px 0px;text-align:center;text-decoration:none;font-size:12pt;color:#000000;display:inline-block;border-radius:3px">Register</a>
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:65]:
```suggestion
an underrepresented demographic in the computational molecular sciences.
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:68]:
```suggestion
the [greater Phoenix area](https://en.wikipedia.org/wiki/Phoenix_metropolitan_area).
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:71]:
```suggestion
our travel agency*, so as an awardee you will have to be able to
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:74]:
```suggestion
Bursary recipients will be notified by May 30. 
```
The timeline works for me (though as a note, May 27 is a Mon, and May 28 is a Tues). I am only changing the date since we say _by_ May XX. That way, we have May 28 to review/decide, and can send notifications out _on_ May 29.

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [reply _posts/2024-05-14-ASUworkshop.md:74]: The registration form currently asks about attendance based on financial support in a slightly different way. After asking if people need support for hotel and/or travel, it asks estimated total costs (for travel) as well as amount of $ that could be obtained elsewhere (e.g., university travel awards, PI grants, etc.). It asks generally if they'd be unable to attend without the support asked for, so I think we could make decisions to answer the above.

Happy to change it to the way described above (or feel free to do so at https://docs.google.com/forms/d/1-_iVRym3oOxMkozB2TnBPWVUdk6ALJpAVt2fpNV9s8M/edit), but was just worried about people having fatigue answering so many questions

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [reply _posts/2024-05-14-ASUworkshop.md:79]: I agree that we should just keep it simple and have one deadline.

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:148]:
```suggestion
rare, so it is important that you bring a hat for sun protection and carry
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:151]:
```suggestion
Avoid hiking in the desert during the hot portion of the day (10am - 6pm), as it
```

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [inline _posts/2024-05-14-ASUworkshop.md:154]: ### Who to Contact

If you have any questions or special requests related to this workshop, you may [contact the organizing committee](mailto:workshops@mdanalysis.org).

File: _posts/2024-05-14-ASUworkshop.md
Comment (jennaswa) [reply _posts/2024-05-14-ASUworkshop.md]: Based on our experience with the hybrid TYC workshop, it was very manageable to have 1-2 folks answering questions from participants online (we had ~20 people). Everyone online kept their videos off and didn't take us up on offers to unmute and ask questions, so we can probably accommodate at least another 40 online. I would say we don't have to limit it now, and can make a decision based on the number of applications we receive, which sessions we decide to make hybrid, and how many instructors we have able to help with the hybrid portion during each session.

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [reply _posts/2024-05-14-ASUworkshop.md:72]: The reg form looks good, thanks @jennaswa . I agree with you that it provides the information we need.

File: _posts/2024-05-14-ASUworkshop.md
Comment (orbeckst) [reply _posts/2024-05-14-ASUworkshop.md]: I appreciated the detail to semantics. 

I left it as "on May 29 or earlier" to convey that we are keeping it tight. I'd like people to start looking out for notification starting May 29. 

The reg form also contains May 29.

**Review by orbeckst — COMMENTED** [review]
As far as I am concerned, the page and registration form are ready.

@jennaswa @fiona-naughton please have a final look and if you're happy give your approval, please, so that we can lauch either today or early tomorrow.

@armcdona I added you at the bottom of the page as part of the org team. Please let me know if that's ok.

**Comment by orbeckst** [comment]
I'll merge once org committee approves.

I'll make a separate issue to keep track of dissemination/advertising.

**Comment by armcdona** [comment]
Looks good to me!  Let me know when your event page/registration is live and we can also advertise it.

**Review by fiona-naughton — APPROVED** [review]
lgtm - thanks!

**Review by jennaswa — APPROVED** [review]
LGTM, too - thanks, Oliver!

**Pull request merged**
